### DISCUSSION TITLE
GDPR Ready

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#604] [General help] [answered]
On your website there is : 
Deepgram is GDPR ready. We provide information to our customers to help them understand how features and functionality of our platform may affect their GDPR compliance obligations.

What do I have to do to be GDPR compliant in my Saas ?

> [Posted by towny1053](https://discord.com/channels/1108042150941294664/1206905774039633950/1206905774039633950)

**Comment by team-deepgram** [comment, marked as answer]
I was able to get some information for you...

On-premises deployment is not actually required for customers to achieve GDPR compliance with Deepgram. Typically, customers need a Data Processing Agreement (DPA) with specific clauses, such as the Standard Contractual Clauses (SCCs), which Deepgram's standard DPA includes.

Some EU customers may have an additional requirement that all data must be processed within the EU. While this is not a legal requirement, some businesses demand it for compliance. For these customers, on-premises deployment is currently the only option.



> [Posted by sandra_rodgers](https://discord.com/channels/1108042150941294664/1206905774039633950/1207038788103176262)

**Reply by vr4content** [reply]
Hi, my company (european company) has a VR App using voice transcriptions, but to add deepgram to our STT providers list our legal advisor says we still need a signed DPA agreement. Other platforms we are using for STT and other services, they offer automatised process to request and get those certificates, is Deepgram offering something similar? or is there a specific email I can ask for this?

**Comment by carlesatm** [comment]
Hi Deepgram team,

I'm reaching out to request a Data Processing Agreement (DPA) with Deepgram, including the Standard Contractual Clauses (SCCs) required for GDPR compliance.

Account details:

Email: [carlesarnautm@gmail.com](mailto:carlesarnautm@gmail.com)
Project ID: 47ab0cd8-a979-4727-839e-5c850c40d521
Could you please share the DPA document or let me know the process to execute it?

Thank you!

**Reply by naomi-deepgram** [reply]
hiya! could you please email me directly at `naomi.carrigan@deepgram.com`?
